src/sinks/elasticsearch/sink.rs

```diff
@@ -30,17 +27,6 @@ pub struct PartitionKey {
     pub bulk_action: BulkAction,
 }
 
-pub struct BatchedEvents {
-    pub key: PartitionKey,
-    pub events: Vec<ProcessedEvent>,
-}
-
-impl ByteSizeOf for BatchedEvents {
-    fn allocated_bytes(&self) -> usize {
-        self.events.size_of()
-    }
-}
-
```

Comment:
I noticed this was unused while looking into an ES thing.

Reply:
:fry: if we're being particular... definitely a separate PR for this

Reply:
It's dead code though... would a compromise, be to run integration tests , since there is an ES integration test?
It's one thing if behavior is changed, or even just simple logic is changed. But this is pretty clearly dead code.

Reply:
🤷 it's more about being unrelated to the PR/commit, rather than breaking - I'm not going to block the PR on it though

Reply:
bahumbug. I see your point. I guess its really not <that> much effort to open a separate PR.